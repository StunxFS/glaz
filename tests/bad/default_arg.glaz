fn x(_k: bool = true, _a: i32) {
}

fn main() {
}
