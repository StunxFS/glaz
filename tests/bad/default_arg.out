tests/bad/default_arg.glaz:1:7: error: arguments with default values must go at the end
     | 
   1 | fn x(_k: bool = true, _a: i32) {
     |      ^^^^^^^^^^^^^^^
glaz error: could not compile `default_arg`, aborting due to previous error.
